src/test/java/org/openrewrite/java/testing/junit5/AddTestcontainersAnnotationsTest.java

```diff
@@ -0,0 +1,322 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Moderne Source Available License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://docs.moderne.io/licensing/moderne-source-available-license
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.testing.junit5;
+
+import static org.openrewrite.java.Assertions.java;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+public class AddTestcontainersAnnotationsTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipe(new AddTestcontainersAnnotations())
+          .parser(JavaParser.fromJavaVersion().classpath("junit", "testcontainers", "junit-jupiter"));
```

Comment:
Since there's more than one artifact with these prefixes you'll likely want to use numerical suffixes:
```suggestion
          .parser(JavaParser.fromJavaVersion().classpath("junit-4", "testcontainers-1", "junit-jupiter-1"));
```